docs/en/Running-Alluxio-Fault-Tolerant.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: global
-title: Alluxio Standalone with Fault Tolerance
-nickname: Alluxio Standalone with Fault Tolerance
+title: Alluxio on a Cluster with High Availability
+nickname: Alluxio on a Cluster with HA
```

Comment:
I prefer `Alluxio with High Availability` or (`Alluxio on a Cluster with High Availability` if that fits). I think the cluster portion is implicit.